facdb/sql/_create_facdb.sql

```diff
@@ -12,9 +12,9 @@ SELECT
     facdb_spatial.bbl::bigint as bbl,
     facdb_spatial.commboard as cd,
     facdb_spatial.nta,
-    facdb_spatial.council,
-    facdb_spatial.schooldist,
-    facdb_spatial.policeprct,
+    LPAD(facdb_spatial.council,2,'0') as council,
```

Comment:
this should happen in 
https://github.com/NYCPlanning/db-facilities/blob/5f88854eed5dbc2bea2ec47b277f804825eef6c8/facdb/sql/_create_facdb_spatial.sql#L41-L59
similar to bin formatting, all other districts formating should be done here ^